Specs/Scene/ModelExperimental/Model3DTileContentSpec.js

```diff
@@ -464,73 +924,431 @@ describe(
+          tileset.pointCloudShading.attenuation = true;
+          tileset.pointCloudShading.geometricErrorScale = 1.0;
+          tileset.pointCloudShading.maximumAttenuation = 1;
+          tileset.pointCloudShading.baseResolution = undefined;
+          tileset.maximumScreenSpaceError = 16;
+          expect(scene).toRenderPixelCountAndCall(function (pixelCount) {
+            expect(pixelCount).toEqual(noAttenuationPixelCount);
+          });
+        });
+      });
+
+      it("modulates attenuation using the baseResolution parameter", function () {
+        return attenuationTest(function (scene, tileset) {
+          // pointCloudNoColorUrl is a single tile with GeometricError = 0,
+          // which results in default baseResolution being computed
+          tileset.pointCloudShading.attenuation = true;
+          tileset.pointCloudShading.geometricErrorScale = 1.0;
+          tileset.pointCloudShading.maximumAttenuation = undefined;
+          tileset.pointCloudShading.baseResolution = 0.2;
+          tileset.maximumScreenSpaceError = 16;
+          expect(scene).toRenderPixelCountAndCall(function (pixelCount) {
+            expect(pixelCount).toEqual(noAttenuationPixelCount);
+          });
+        });
+      });
+
+      it("modulates attenuation using the geometricErrorScale parameter", function () {
+        return attenuationTest(function (scene, tileset) {
+          tileset.pointCloudShading.attenuation = true;
+          tileset.pointCloudShading.geometricErrorScale = 0.2;
+          tileset.pointCloudShading.maximumAttenuation = undefined;
+          tileset.pointCloudShading.baseResolution = 1.0;
+          tileset.maximumScreenSpaceError = 1;
+          expect(scene).toRenderPixelCountAndCall(function (pixelCount) {
+            expect(pixelCount).toEqual(noAttenuationPixelCount);
+          });
+        });
+      });
+
+      it("attenuates points based on geometric error in 2D", function () {
+        return attenuationTest(function (scene, tileset) {
+          scene.morphTo2D(0);
+          tileset.pointCloudShading.attenuation = true;
+          tileset.pointCloudShading.geometricErrorScale = 1.0;
+          tileset.pointCloudShading.maximumAttenuation = undefined;
+          tileset.pointCloudShading.baseResolution = undefined;
+          tileset.maximumScreenSpaceError = 16;
+          expect(scene).toRenderPixelCountAndCall(function (pixelCount) {
+            expect(pixelCount).toBeGreaterThan(noAttenuationPixelCount);
+          });
+        });
+      });
     });
 
-    it("picks from b3dm batch table", function () {
-      setCamera(centerLongitude, centerLatitude, 15.0);
-      return Cesium3DTilesTester.loadTileset(scene, withBatchTableUrl).then(
-        function (tileset) {
+    describe("glTF", function () {
+      beforeEach(function () {
+        setCamera(centerLongitude, centerLatitude, 100.0);
+      });
+
+      it("resolves readyPromise with glb", function () {
```

Comment:
is there any reason these stayed in but the `"resolves readyPromise with b3dm"` and `"resolves readyPromise with i3dm"` tests got removed?

Reply:
this was one of the cases where we wanted to consolidate the test to only use one instead of checking every format. So I just kept the glTF ones.

Reply:
okay thats fine. I was just confused by the specificity between glTF and glb